src/data-card-advanced.tsx

```diff
@@ -0,0 +1,463 @@
+
+            <div className={styles.actions} style={{flexDirection, alignItems}}>
+                {hasButton && (
+                    <div
+                        aria-hidden="true"
+                        onClick={(event) => event.stopPropagation()}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        style={{marginTop: '16px', marginRight: '16px', position: 'relative', zIndex: '2'}}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div style={{display: 'flex', flexDirection: 'row', marginTop, flexWrap: 'wrap'}}>
+                    {hasFooterImage && (
+                        <div
+                            style={{marginRight: '16px', zIndex: '0'}}
+                            className={sprinkles({alignItems: 'center', display: 'flex'})}
+                        >
+                            {footerImage}
+                        </div>
+                    )}
+
+                    {hasFooterText && (
+                        <div style={{maxWidth}} className={styles.footerText}>
+                            <Text2 truncate={footerTextLinesMax} regular>
+                                {footerText}
+                            </Text2>
+                        </div>
+                    )}
+                </div>
+                {hasButtonLink && (
+                    <div
+                        aria-hidden="true"
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        onClick={(event) => event.stopPropagation()}
+                        style={{marginTop, position: 'relative', zIndex: '2'}}
+                    >
+                        {buttonLink}
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+};
+
+type slotsTypeof = typeof StackingGroup;
+
+type textAs = 'h1' | 'h2' | 'h3' | 'h4';
+
+interface DataCardAdvancedProps {
+    cardOnPress?: () => void;
+
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    pretitleAs?: textAs;
+    title?: string;
+    titleLinesMax?: number;
+    titleAs?: textAs;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+
+    slots?: Array<RendersNullableElement<slotsTypeof>>;
+    small?: boolean;
+
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+
+    dataAttributes?: DataAttributes;
+    actions?: Array<CardAction>;
+    'card-aria-label'?: string;
+    'touchable-aria-label'?: string;
```

Comment:
why is this needed? shouldn't be the card aria label enough?

Reply:
yes, i will remove the touchable-aria-label